### PR TITLE
connect to API

### PR DESCRIPTION
**Pull request by disc3110**
- Get books from API
- Send books to API
- Delete books from API

[live page](https://disc3110.github.io/bookstore/)

### CODE REVIEW

**Comment by disc3110** [comment]
I have to refresh the page to see the changes from the API. I couldn't manage to solve that. Any suggestions?. I tried to do it with useEffect

File: src/redux/books/books.js [hunk 1]
```diff
@@ -1,24 +1,71 @@
...
+    if (response.ok) {
+      dispatch({
+        type: ADD_BOOK,
+        payload,
+      });
+    }
+  } catch (error) {
+    console.log(error);
+  }
+};
```
Comment (mricanho) [inline src/redux/books/books.js:28]: - To get the changes automatically, you need to refactor this, adding a reducer in which you set the initialState and then calling the initialState or "state" once the response is ok.

File: src/redux/books/books.js [hunk 2]
```diff
@@ -1,24 +1,71 @@
...
-export const addBook = (payload) => ({
-  type: ADD_BOOK,
-  payload,
-});
+export const addBook = (payload) => async (dispatch) => {
+  console.log('ive been called');
+  console.log(payload);
+  const { item_id, title, category } = payload;
+  try {
+    const response = await axios.post(baseURL, { item_id, title, category });
```
Comment (mricanho) [inline src/redux/books/books.js:18]: - Here you are only sending the data, and that's it, you need to call something after you achieve this.

**Review by mricanho — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @disc3110  🔥 

The hardcore requirements are met, your app is connected to the API, but I strongly suggest that you work and refactor the function I appoint, because in the next milestone will be mandatory👍🏽 

awesome-job

Now you can merge this branch 🚀 

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**